lib/galaxy/model/migrations/util.py

```diff
@@ -0,0 +1,6 @@
+from alembic import op
+
+
+def drop_column(table, column):
+    with op.batch_alter_table("workflow") as batch_op:
+        batch_op.drop_column("source_metadata")
```

Comment:
I guess you meant this :wink: 
```suggestion
    with op.batch_alter_table(table) as batch_op:
        batch_op.drop_column(column)
```

Reply:
Whoops. That was dumb. Thank you, @davelopez !